tests/test_core.py

```diff
@@ -204,102 +175,63 @@ def test_i64_precise(self):
   def test_i64_llabs(self):
     Settings.PRECISE_I64_MATH = 2
 
-    test_path = path_from_root('tests', 'core', 'test_i64_llabs')
-    src, output = (test_path + s for s in ('.in', '.out'))
-
-    self.do_run_from_file(src, output)
+    self.do_run_in_out_file_test('tests', 'core', 'test_i64_llabs')
 
   def test_i64_zextneg(self):
-    test_path = path_from_root('tests', 'core', 'test_i64_zextneg')
-    src, output = (test_path + s for s in ('.in', '.out'))
-
-    self.do_run_from_file(src, output)
+    self.do_run_in_out_file_test('tests', 'core', 'test_i64_zextneg')
 
   def test_i64_7z(self):
-    test_path = path_from_root('tests', 'core', 'test_i64_7z')
+    self.do_run_in_out_file_test('tests', 'core', 'test_i64_7z')
```

Comment:
Looks like this is still an issue, i.e., this commit has not been fixed and would break bisection?